ethanschreur/kids-krafts
# Customers can select a pickup date and time

**What does this PR do?**
- This pull request allows customers to select a pickup date and time in an "order detail" page

**Description of the task to be completed** 
- Create an "order detail" page
- Write helper functions to help create the calendar
- Display a calendar that covers the options for pickup dates which should be two weeks around the start of each month.
- Each day has options for AM and PM
- Customers can only select one option on the calendar

**How should this  be tested**
- (On Windows) Run `. venv/scripts/activate` 
- `python run.py`
- Go to http://127.0.0.1:5000/order_details
- View the calendar with the two weeks around the start of each month
- Make sure you can only select one option
